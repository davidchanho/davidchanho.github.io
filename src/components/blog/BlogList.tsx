import BlogPost from 'components/blog/BlogPost'
import Section from 'components/section/Section'
import React from 'react'
import { ListGroup } from 'react-bootstrap'
import blog1 from '../../blog/blog_1.md'

function BlogList() {
    return (
        <Section name='blog' header='blog'>
            <ListGroup>
                <BlogPost file={blog1} />
            </ListGroup>
        </Section>
    )
}

export default BlogList
