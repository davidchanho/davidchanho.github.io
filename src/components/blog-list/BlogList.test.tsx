import React from "react";
import renderer from "react-test-renderer";
import BlogList from ".";

describe("BlogList", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<BlogList />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
