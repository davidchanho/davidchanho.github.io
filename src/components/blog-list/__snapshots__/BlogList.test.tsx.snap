// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogList renders correctly 1`] = `
<div
  className="sc-bdfBwQ fKNFrQ"
  name="blog"
>
  <h3
    className="text-center text-capitalize mb-5"
  >
    <u>
      blog
    </u>
  </h3>
  <div
    className="container"
  >
    <div
      className="list-group"
      onKeyDown={[Function]}
    >
      <div
        className="border-0 list-group-item"
        data-rb-event-key={null}
        disabled={false}
        onClick={[Function]}
      />
    </div>
  </div>
</div>
`;
